src/parser.js

```diff
@@ -462,6 +463,14 @@ export default class Parser extends Stream {
 
               currentUri.dateTimeString = entry.dateTimeString;
               currentUri.dateTimeObject = entry.dateTimeObject;
```

Comment:
Maybe we should rename it to the `programDateTime` and `programDateTimeObject` for consistency with `programDateTime`

Reply:
Maybe parsing both `dateTimeString` and `dateTimeObject` is unnecessary. Removed `dateTimeObject` and renamed `dateTimeString` to `programDateTimeString`